src/ontology/mondo-ingest.Makefile

```diff
@@ -32,13 +32,16 @@ dependencies:
 ### General ########################
 ####################################
 %.db: %.owl
+	@rm $*.db
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f $*-relation-graph.tsv.gz
 	RUST_BACKTRACE=full semsql make $*.db -P config/prefixes.csv
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f $*-relation-graph.tsv.gz
+	@test -f $*.db || (echo "Error: File not found!" && exit 1)
```

Comment:
The above pipeline (`semsql make`) does not fail hard, so this goal (`%.db`) always succeeds. This way, we make it fail (by first deleting the db file, and after the pipeline has finished checking if it is there.

Reply:
Hmm, that's unfortunate that it doesn't fail hard when it fails. Should I make an issue about that in the 
Your `test -f` solution is good.

However, `rm $*.db` will fail if you run this goal and the file doesn't already exist, causing `make` to error out. `rm -f $*.db` should solve.

Reply:
Great suggestion, merged.